airgun/views/puppet_environment.py

```diff
@@ -55,6 +55,14 @@ class ImportPuppetEnvironmentView(BaseLoggedInView, SearchableViewMixin):
     obsolete = Text("//a[contains(@data-original-title,'obsolete')]")
     update = Text("//input[@name='commit']")
     cancel = Text("//a[@data-id='aid_environments']")
+    table = SatTable(
+        locator='.//table',
+        column_widgets={
+            'Environment': Text('//a[starts-with(\
```

Comment:
How many links are there in 'Environment' column? If that's the only one - just use `./a` instead. And btw use relative path (starting with `.`).